scipy/io/wavfile.py

```diff
@@ -278,10 +301,16 @@ def read(filename, mmap=False):
         else:
             fid.seek(0)
 
-    return fs, data
+    result = [fs, data]
+    if return_cues:
+        result.append(cue)
+        result.append(cuelabels)
+    if return_pitch:
+        result.append(pitch)
+    return tuple(result)
```

Comment:
I agree. Is the current preference to use a `namedtuple` or a `Bunch` for this sort of thing?